src/main/java/uk/gov/hmcts/darts/common/entity/EventEntity.java

```diff
@@ -23,17 +26,12 @@ public class EventEntity {
     @SequenceGenerator(name = "moj_eve_gen", sequenceName = "moj_eve_seq", allocationSize = 1)
     private Integer id;
 
-    @Column(name = "moj_crt_id")
-    private Integer courthouseId;
-
-    @Column(name = "moj_evt_id")
-    private Integer eventTypeId;
-
     @Column(name = "r_event_object_id", unique = true, length = 16)
     private String legacyObjectId;
 
-    @Column(name = "c_event_id")
-    private Integer eventId;
+    @ManyToOne()
+    @JoinColumn(name = "moj_evt_id")
+    private EventTypeEntity eventId;
```

Comment:
Ah, I see there is a column in v26 `c_event_id                 NUMERIC`
https://tools.hmcts.net/confluence/display/DMP/Event+API+LLD

That's probably where the confusion arised - maybe it is `legacyEventId` ?